down

```diff
@@ -7,18 +7,57 @@ import sqlite3
 import os
 import argparse
 import itertools
+import time
 
 import requests
+import threading
 
-
-conn = sqlite3.connect('data.db', timeout=sys.float_info.max)
+conn = sqlite3.connect('data.db', timeout=sys.float_info.max, check_same_thread=False)
```

Comment:
sqlite3 is not thread safe

consider using locks: https://stackoverflow.com/a/26630550